pulp/apis/cplex_api.py

```diff
@@ -339,6 +339,8 @@ def buildSolverModel(self, lp):
                 lp.solverModel.objective.set_sense(
                     lp.solverModel.objective.sense.maximize
                 )
+            if lp.objective is None:
+                raise PulpSolverError("No objective set")
```

Comment:
in theory it's possible to not set an objective function afaik and just model a constraint satisfaction problem. Isn't that the case?